.storybook/main.ts

```diff
@@ -77,5 +77,8 @@ const config: StorybookConfig = {
 
     reactDocgen: "react-docgen-typescript",
   },
+  features: {
+    experimentalRSC: true,
```

Comment:
Im adding this experimental flag for future stories that may use RSC but it didn't work for the home hero. That is why Im removing it for now until we have a solution.

ref https://storybook.js.org/docs/get-started/frameworks/nextjs#react-server-components-rsc